sunnypilot/selfdrive/controls/lib/nnlc/nnlc.py

```diff
@@ -196,7 +196,7 @@ def update_feed_forward(self, CS, params, setpoint, measurement, calibrated_pose
                     adjusted_future_times]
     past_lateral_accels_desired = [self.lateral_accel_desired_deque[min(len(self.lateral_accel_desired_deque) - 1, i)]
                                    for i in self.history_frame_offsets]
-    future_planned_lateral_accels = [np.interp(t, ModelConstants.T_IDXS[:CONTROL_N], self.model_v2.acceleration.y) for t in
+    future_planned_lateral_accels = [np.interp(t, ModelConstants.T_IDXS, self.model_v2.acceleration.y) for t in
```

Comment:
@twilsonco I had to make this change as it was crashing with `fp and xp are not of the same length.`

Reply:
Yeah, the old "numpy fast" interp function could handle different lengths.  The intent was to limit the farthest future model lateral acceleration value that could be returned, so it's better to instead take [:N] from both arrays.